### ISSUE TITLE
Ruby local variable color is wrong

### ISSUE THREAD

**Issue by alexandru-calinoiu** [jkaving/intellij-colors-solarized#114]
After importing the .jar file in RubyMine 8.0.3 the local variable name is set to 003C5A in solarize dark, I think it should be B58900.

**Comment by jkaving** [comment]
Thanks for noticing!

It should be fixed now - please import the updated `settings.jar` file.

**Comment by alexandru-calinoiu** [comment]
If you don't mind me asking where did you came up with RUBY_LOCAL_VAR_ID I wanted to submit a pr yesterday but I couldn't figure out how to get the option name.

**Comment by jkaving** [comment]
I downloaded and installed RubyMine (because I didn't have it installed - I mainly use IntelliJ IDEA).
After starting it I installed the `settings.jar` file from this repo.

Then I went into the preferences and selected Editor -> Color & Fonts -> Ruby:
<img width="873" alt="screenshot of rubymine 2016-01-15 06-49-31" src="https://cloud.githubusercontent.com/assets/725090/12346390/24be8500-bb54-11e5-8efa-d24a6af91822.png">

I selected the "Local variable" setting and changed to color to the correct one.

If I remember correctly I then diff:ed the contents of the color scheme file from this repo with the updated one that was now stored in RubyMine's color settings directory. Something like:
```
> diff intellij-colors-solarized/Solarized\ Light.icls ~/Library/Preferences/RubyMine80/colors/Solarized\ Light.icls
```

Unfortunately I don't think the color id's are documented somewhere, so the only way I know for finding them is looking at the color scheme file that the application saves in the settings directory.
